scripts/monorepo/src/utils.js

```diff
@@ -9,9 +9,20 @@ const TEN_MEGABYTES = 1024 * 10000;
 /**
  * Gets nx project metadata
  * @param {string} projectName - package name
+ * @param {ReturnType<typeof import('@nx/devkit').getProjects>=} allProjects - all workspace projects
  * @returns {import('@nx/devkit').ProjectConfiguration}
  */
-function getProjectMetadata(projectName) {
+function getProjectMetadata(projectName, allProjects) {
```

Comment:
Just making sure that this change is intended since I don't see this new API being used anywhere else in this PR

Reply:
excellent point, I removed/replaced the API completely https://github.com/microsoft/fluentui/pull/31008/commits/5f8196b391e0c3994671b8d2585a823c8b045661